modules/openapi-generator/src/main/resources/rust-server/Cargo.mustache

```diff
@@ -17,7 +17,7 @@ conversion = ["frunk", "frunk_derives", "frunk_core", "frunk-enum-core", "frunk-
 # Common
 chrono = { version = "0.4", features = ["serde"] }
 futures = "0.1"
-swagger = "2.2"
+swagger = "4.4"
```

Comment:
This version doesn't exist, hence why the build is failing.

The other problem is that the Swagger v4 requires Hyper 0.12, whereas this code is still using 5.0.x